### PR TITLE
INTYGFV-15395: Added SRS panel

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: packages/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx [hunk 1]
```diff
@@ -1,70 +1,78 @@
...
-        <div data-tip={questionsNotAvailablePanelActive.description}>
-          <p>{questionsNotAvailablePanelActive.name}</p>
-        </div>
-      )
-
-      tabsContentArray.push(<QuestionNotAvailablePanel />)
+  const getPanel = (type: ResourceLinkType) => {
+    switch (type) {
+      case ResourceLinkType.FMB:
+        return <FMBPanel></FMBPanel>
```
Comment (SamuelTilly) [inline packages/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx:53]: Very minor `<FMBPanel />`

File: packages/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx [hunk 2]
```diff
@@ -1,70 +1,78 @@
...
-    )
+    resourceLinksForTabs.forEach((type) => {
+      const link = resourceLinks.find((l) => l.type === type)
+      if (link) {
+        tabsArray.push(getTabFromLink(link, getIcon(link.type)))
+        tabsContentArray.push(getPanel(link.type))
+      }
+    })
 
+    tabsArray.push(getTab('Om intyget', 'Läs om intyget'))
```
Comment (SamuelTilly) [inline packages/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx:75]: Good refactoring here, but I think we can take it a bit further.

We can create a list with resource links for tabs:
```typescript
const resourceLinksForTabs = [
  ResourceLinkType.SRS,
  ResourceLinkType.FMB,
  ResourceLinkType.QUESTIONS,
  ResourceLinkType.QUESTIONS_NOT_AVAILABLE,
]
const availableTabs = resourceLinks.filter(({ type }) => resourceLinksForTabs.includes(type))
```
Then we can remove the `getTabs` function and simply do:

```typescript
return (
  <Tabs
    selectedTabIndex={selectedTabIndex}
    setSelectedTabIndex={handleTabChange}
    tabs={[
      ...availableTabs.map(({ type, name, description }) => getTab(name, description, getIcon(type))),
      getTab('Om intyget', 'Läs om intyget'),
    ]}
    tabsContent={[...availableTabs.map(({ type }) => getPanel(type)), <AboutCertificatePanel key={'about'} />]}
  />
)
```

**Review by SamuelTilly — CHANGES_REQUESTED** [review]
Very nice work! Left some minor suggestions

File: packages/webcert/src/components/srs/SRSPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      return ''
+    }
+
+    const result = diagnosisCodes.find((code) => diagnosisListValue?.list[0].code === code)
+    return result ? result : ''
+  }
+
+  const hasSupportedDiagnosisCode = (): boolean => {
+    return getSupportedDiagnosisCode().length > 0
+  }
```
[the remark this answers is not in the document]
Comment (mhornfeldt) [reply packages/webcert/src/components/srs/SRSPanel.tsx:38]: I think it's better with the function since the condition is used in several places

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**